tasks/db_deploy/migrate_db.py

```diff
@@ -26,7 +26,12 @@ def perform_migration(current_schema_version: int, config: Dict[str, str]) -> No
         # Run migrations from version 1 to the latest version
         # in this case version 2 is the latest so we set the latest version
         # flag to True
-        migrate_versions_1_to_2(config, True)
+        migrate_versions_1_to_2(config, False)
+        current_schema_version = 2
+
+    if current_schema_version == 2:
+        migrate_versions_2_to_3(config, True)
+        current_schema_version = 3
```

Comment:
are adding these `current_schema_version = 2` and `current_schema_version = 3` required?

Reply:
The former yes, the later no. Following the existing model, unless the current_schema_version is updated only one of the migrations max will occur.